### PR TITLE
Remove input elements

### PR DESCRIPTION
**Pull request by tabascq**
Input elements make styling unpredictable so I am taking them out.

### CODE REVIEW

File: puzzle.css [hunk 1]
```diff
@@ -21,11 +21,14 @@
     font-weight: bold;
     font-family: monospace;
     border: none;
-    color: transparent;
-    text-shadow: 0 0 0 black, -2px -2px 0 #fff, 0 -2px 0 #fff, 2px -2px 0 #fff, 2px 0 0 #fff, 2px 2px 0 #fff, 0 2px 0 #fff, -2px 2px 0 #fff, -2px 0 0 #fff;
+    color: black;
+    text-shadow: -2px -2px 0 #fff, 0 -2px 0 #fff, 2px -2px 0 #fff, 2px 0 0 #fff, 2px 2px 0 #fff, 0 2px 0 #fff, -2px 2px 0 #fff, -2px 0 0 #fff;
+    user-select: none;
+    cursor: pointer;
+    text-transform: uppercase;
```
Comment (jenna-h) [inline puzzle.css:28]: Todo for later: create a setting that allows puzzles to have lowercase letters?

OK as is for this PR

File: puzzle.js [hunk 2]
```diff
@@ -813,25 +824,25 @@ function PuzzleEntry(p) {
...
-                cell.value = this.translate(shapeCh, shapeReplacements);
+                text.innerText = this.translate(shapeCh, shapeReplacements);
                 td.classList.add("given");
                 if (unselectableGivens) { td.classList.add("unselectable"); }
             }
 
             if (!td.classList.contains("unselectable")) {
-                cell.addEventListener("keydown",  e => { this.keyDown(e); });
-
+                td.tabIndex = -1;
```
Comment (jenna-h) [inline puzzle.js]: This makes it so we can't tab into the interactive elements, or tab from one interactive element to the next.

Can we make the very first td of every puzzle have tabIndex = 0? The rest should be -1

File: puzzle.css [hunk 3]
```diff
@@ -21,11 +21,14 @@
...
 }
```
Comment (jenna-h) [inline puzzle.css:30]: I found the focus border hard to see. Can we add

```
.puzzle-entry td:focus .text {
    box-shadow: 0px 0px 3px 3px Highlight;
    box-shadow: 0px 0px 3px 3px -webkit-focus-ring-color;
}
```

File: puzzle.js [hunk 4]
```diff
@@ -684,10 +696,9 @@ function PuzzleEntry(p) {
         // Copy any text inside the td. This includes text inside divs within the td.
         copyTd.innerText = inputTd.innerText;
         // If the td has a "value", overwrite the innertext.
-        // TODO(jhimawan): fix after no-input refactor.
-        const input = inputTd.getElementsByTagName('input')[0];
-        if (input?.value) {
-            copyTd.innerText = input.value;
+        const text = inputTd.querySelector('text');
```
Comment (jenna-h) [inline puzzle.js:700]: Should be 
```
const text = inputTd.querySelector('.text');
if (text?.innerText) {
    copyTd.innerText = text.innerText;
}
```

File: puzzle.js [hunk 5]
```diff
@@ -282,8 +293,8 @@ function PuzzleEntry(p) {
         else if (e.keyCode == 40) { this.move(e.target, 1, 0); } // down
         else if (e.keyCode == 32) { // space
             this.dx = 1 - this.dx; this.dy = 1 - this.dy;
-            if (e.currentTarget.parentElement.classList.contains("given-fill")) return;
-            if (this.options["data-fill-cycle"]) { this.currentFill = this.cycleClasses(e.target.parentElement, this.fillClasses, e.shiftKey); }
+            if (e.currentTarget.classList.contains("given-fill")) return;
```
Comment (jenna-h) [inline puzzle.js:297]: In the keyCode == 32 block, I found that adding

```
// Reset focus after dx, dy change.
e.target.blur();
e.target.focus();
```

Allowed the dx, dy changes to propagate to the crossword highlighting.

It seems a little hacky, so I'm open to other suggestions :)

File: puzzle.js (hunk 2 above)
Comment (tabascq) [reply puzzle.js]: I'll do this but TBH it doesn't seem to make any difference

File: puzzle.css (hunk 3 above)
Comment (tabascq) [reply puzzle.css:30]: Hmm, once this change is in, the above change is much easier to see :-)

File: puzzle.js (hunk 4 above)
Comment (tabascq) [reply puzzle.js:700]: Indeed oops

File: puzzle.css (hunk 3 above)
Comment (jenna-h) [reply puzzle.css:30]: I'm sorry, I'm not quite sure what you mean?

Comment (tabascq) [reply puzzle.css:30]: the tabindex=0 didn't seem to do anything but now it does

File: puzzle.css (hunk 1 above)
Comment (tabascq) [reply puzzle.css:28]: low pri for me for now, will revisit if/when someone needs this

**Comment by tabascq** [comment]
Thanks for the suggestions!

**Pull request merged**